Polygon.Net/API/ReferenceApi.cs

```diff
@@ -168,5 +171,23 @@ public async Task<List<MarketHoliday>> GetMarketHolidaysAsync()
 
             return JsonConvert.DeserializeObject<List<MarketHoliday>>(contentStr);
         }
+
+        /// <summary>
+        /// Get the ticker types from Polygon for the asset class and locale.
+        /// </summary>
+        /// <param name="assetClass">One of the following values <see cref="AssetClass"/>.</param>
+        /// <param name="locale">One of the values "us" or "global".</param>
+        /// <returns></returns>
+        public async Task<PolygonResponse<TickerType>> GetTickerTypesAsync(string assetClass = default, string locale = default)
+        {
+            assetClass ??= AssetClass.Stocks;

```

Comment:
Why not just make this the default param instead of default? That way no need for a null check.

Reply:
I need a compile variable here; with a class I can make it happen.